### PR TITLE
Chrome for Android 70 supports webauthn

### PR DESCRIPTION
**Pull request by bdewater**
https://blog.chromium.org/2018/09/chrome-70-beta-shape-detection-web.html

### CODE REVIEW

**Comment by HM100** [comment]
No adding new browser versions, please?
@Fyrd What's wrong with Travis on your repository?

File: features-json/webauthn.json [hunk 1]
```diff
@@ -296,7 +296,8 @@
       "46":"n"
     },
     "and_chr":{
-      "69":"n"
+      "69":"n",
+      "70":"y"
```
Comment (HM100) [inline features-json/webauthn.json]: This PR is invalid because it adds new browser version

Comment (bdewater) [reply features-json/webauthn.json]: So should it just be `"70":"y"` instead?

Comment (HM100) [reply features-json/webauthn.json]: Nothing. Just remove it.

Comment (bdewater) [reply features-json/webauthn.json]: I'm confused, how does this help in signalling that a soon-to-be-released version has support? That's what I was trying to accomplish.

I've run the tests locally (they passed) and I can't find anything in CONTRIBUTING.md about multiple versions being okay for `chrome` but not for `and_chr`? I must be missing something...

Comment (HM100) [reply features-json/webauthn.json]: No. It is for all browsers

Comment (Fyrd) [reply features-json/webauthn.json]: Yeah, currently upcoming versions for Android aren't tracked. If you'd like to mention that it's coming you can add it in the `notes` section.

**Comment by Fyrd** [comment]
@HM100 I can't quite tell what's going on...I can see the PR builds at https://travis-ci.org/Fyrd/caniuse/pull_requests but I don't know why they aren't appearing here on github.

**Comment by PHPGangsta** [comment]
Please add "Chrome for Android" version 70, it has been released on 2018-10-17. It supports webauthn:
https://www.chromestatus.com/feature/5669923372138496

Adding version 70 should boost the usage information of webauthn from 32% to >=60%
https://caniuse.com/#search=webauthn

**Comment by Fyrd** [comment]
Has been updated.

**Pull request closed without merging**